open-company/open-company-web
# use single icon file + provide generic icon component

trello: https://trello.com/c/tZqW1kc8/361-svg-image-issues

Goal: replace most icons with icons loaded from a single cachable file.

___

Review with https://github.com/open-company/open-company-api/pull/37

___

To test: run locally, 

- check that all topics have correct icons. (You may need to re-import sample data.)
- check that topic edit UI has checkboxes + that they work
- check that topic overlay has icons in top right (pencil, close)
